test/model/src/driver/kv_workload_runner_wt.cpp

```diff
@@ -264,6 +264,22 @@ kv_workload_runner_wt::do_operation(const operation::begin_transaction &op)
     return session->session()->begin_transaction(session->session(), nullptr);
 }
 
+/*
+ * kv_workload_runner_wt::do_operation --
+ *     Execute the given workload operation in WiredTiger.
+ */
+int
+kv_workload_runner_wt::do_operation(const operation::breakpoint &op)
+{
+    std::shared_lock lock(_connection_lock);
+    WT_SESSION *session;
+    int ret = _connection->open_session(_connection, nullptr, nullptr, &session);
```

Comment:
The session is never closed. You can try using wiredtiger_session_guard, which automatically closes the session at the end of its scope (please refer to the operation for create_table for an example).